### ISSUE TITLE
Hide setup call in peer wrapper

### ISSUE THREAD

**Issue by g2flyer** [hyperledger-labs/fabric-private-chaincode#141]
Setup should not be a user-visible function and neither does the retrieval of the enclavePK. So our example flows shouldn't have to do ...
```
    try ${PEER_CMD} chaincode invoke -o ${ORDERER_ADDR} -C ${CHAN_ID} -n ${CC_ID} -c '{"Args":["setup", "ercc"]}' --waitForEvent

    try ${PEER_CMD} chaincode query -o ${ORDERER_ADDR} -C ${CHAN_ID} -n ${CC_ID} -c '{"Args":["getEnclavePk"]}'
```
While the second call is anyway not functionally necessary before we do e2e encryption, the former is necessary.   However, we can do that in the peer wrapper as part of the chaincode instantiate interception:  Just call `setup` automatically after a successfull instantiaton. 

To keep namespaces somewhat more seperate between such internal messages and user-space invokcations (and inspired by C compilers), we might call function like this setup `__setup`

**Comment by mbrandenburger** [comment]
I support making `setup` a FPC operation and removing it's user visibility.

Regarding `init` we should omitting it for the moment and focus on other parts until we really have to support it.

**Comment by mbrandenburger** [comment]
Ideally, setup is provided by the Enclave Manager (see new ercc design diagrams) and calls ecc/ or the enclave without leveraging the chaincode interface. This is easy once we have a "custom" chaincode type - or an extended chaincode shim.

**Comment by g2flyer** [comment]
> Regarding `init` we should omitting it for the moment and focus on other parts until we really have to support it.

"outsourced" in separate issue #145 to keep discussions separate

**Comment by g2flyer** [comment]
closed in PR #144

**Comment by g2flyer** [comment]
closed in PR #144
